src/js/components/layer/doc.js

```diff
@@ -0,0 +1,30 @@
+import { schema, PropTypes } from 'react-desc';
+
+export default Layer => schema(Layer, {
+  description: 'A modal overlay.',
+  usage: `import { Layer } from 'grommet';
+  <Layer/>`,
+  props: {
+    align: [
+      PropTypes.oneOf(['center', 'top', 'bottom', 'left', 'right']),
+      'Which direction the layer contents should emanate from.', {
+        defaultProp: 'center',
+      },
+    ],
+    closer: [
+      PropTypes.node,
+      `The node to be used as the close control.
+      If provided, it is the caller's responsibility to listen to events from the node`,
+    ],
+    messages: [
+      PropTypes.object,
+      'Messages to be replaced for Layer.',
```

Comment:
Explain just a bit more here. Also, can we make this a shape and indicate which keys we look for?

Reply:
good idea!!!!